src/onapsdk/msb/k8s/definition.py

```diff
@@ -123,9 +123,9 @@ def get_definition_by_name_version(cls, rb_name: str, rb_version: str) -> "Defin
         return cls(
             definition["rb-name"],
             definition["rb-version"],
-            definition.get("chart-name"),
-            definition.get("description"),
-            definition.get("labels")
+            definition.get("chart-name", None),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are using the `get()` method to retrieve the value for a key from a dictionary. If the key is not present, then `get()` returns the value that is passed in its optional second parameter. If if the key is not present, then `None` is the default return value. Therefore, there is no need to explicitly specify `None` in the second parameter.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)

Similar issue at line numbers 127 and 128.